imi/src/main/java/org/motechproject/nms/imi/domain/FileAuditRecord.java

```diff
@@ -17,7 +17,10 @@
 
     @Field
     @Column(allowsNull = "false")
-    private String status;
+    private Boolean success;
+
+    @Field
+    private String error;
 
     @Field
     private Integer recordCount;
```

Comment:
Are there any validations that can be added for this field?  Can it be null?  Must it be > 0?  Is there a max recordCount